ci-operator/config/openshift/oauth-server/openshift-oauth-server-master.yaml

```diff
@@ -51,14 +51,14 @@ tests:
     make test-unit JUNITFILE=/tmp/artifacts/junit_report.xml --warn-undefined-variables
   container:
     from: test-bin
-- as: e2e-aws
+- as: e2e-gcp
```

Comment:
If you are platform-agnostic, can we name these `e2e`?  Then you can pivot the backing platform as needed without having Prow get confused about the jobs being renamed.  [Docs][1].  Examples: #10152, #10179, #10180, #10377.

[1]: https://github.com/openshift/release/tree/master/ci-operator#rebalancing-tests-among-platforms

Reply:
The platform being targeted seems like a useful detail to expose. prow confusion is temporary, and how often do we really pivot jobs to different platforms?

Reply:
> The platform being targeted seems like a useful detail to expose.

It's still available if you need it.  I don't see why it needs top billing in the job names on preflights for platform-agnostic repositories.

> prow confusion is temporary, and how often do we really pivot jobs to different platforms?

This would be the fourth or fifth repo to pivot in the past month.  By declaring yourself platform agnostic with a generic job name, you create a larger pool of reschedulable jobs that can be shifted to mitigate future capacity issues.